data/hai/unfettered 0 prologue.txt

```diff
@@ -544,7 +544,7 @@ mission "<uhai> 0: Bring Hai"
 			has "language: Hai"
 		<secondgroup> "He does not seem pleased, speaking with contained anger. You are not sure if he is demanding something, or simply complaining."
 			not "language: Hai"
-		<secondgroup> `"Things are going [poorly, worse]. In all my life never has it been more [rough, unsustainable]! Never have there been so many in need! Never so little to harvest, worse [housing, sanitation]... Something needs to be done, and quickly! We, and our way of life, are dying!"`
+		<secondgroup> `"Things are going [poorly, worse]. In all my life never has it been more [rough, unsustainable]. Never have there been so many in need! Never so little to harvest, worse [housing, sanitation]... Something needs to be done, and quickly! We, and our way of life, are dying."`
```

Comment:
I feel the exclamation points are needed to convey.  `...speaking with contained anger` alternate dialogue from above. 
But, like you Hurl, I'm not an expert in writing.